contrib/i2c-tools/template.py

```diff
@@ -0,0 +1,31 @@
+pkgname = "i2c-tools"
+pkgver = "4.3"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_build_args = ["EXTRA=py-smbus"]
+make_install_args = ["sbindir=/usr/bin"] + make_build_args
+hostmakedepends = ["gmake", "python", "python-setuptools"]
+makedepends = ["linux-headers", "python-devel"]
+depends = ["perl"]
+pkgdesc = "Heterogeneous set of I2C tools and I2C library for Linux"
+maintainer = "Subhaditya Nath <sn03.general@gmail.com>"
+license = "GPL-2.0-or-later"
+url = "https://git.kernel.org/pub/scm/utils/i2c-tools/i2c-tools.git"
+source = f"https://mirrors.edge.kernel.org/pub/software/utils/i2c-tools/i2c-tools-{pkgver}.tar.xz"
+sha256 = "1f899e43603184fac32f34d72498fc737952dbc9c97a8dd9467fadfdf4600cf9"
+# no tests
+options = ["!check"]
+
+
+@subpackage(f"{pkgname}-devel")
```

Comment:
please no gratuitous f-strings

Reply:
why tho? :see_no_evil:

Reply:
because it's unnecessary, doesn't achieve anything, and is inconsistent with how every other template is written